db/db.go

```diff
@@ -103,19 +103,20 @@ func CreateTable(cfg Config) error {
 	}
 
 	defer tx.Rollback()
+ /*     No more dropping table as it should create one time only when DB intiated by first server
+        
+        query := "DROP TABLE IF EXISTS tasks CASCADE"
 
-	query := "DROP TABLE IF EXISTS tasks CASCADE"
+        fmt.Println(query)
 
-	fmt.Println(query)
-
-	_, err = tx.Exec(query)
-
-	if err != nil {
-		return err
-	}
-
-	query = "CREATE TABLE tasks ( id SERIAL PRIMARY KEY, completed boolean NOT NULL, priority integer NOT NULL, title text NOT NULL)"
+        _, err = tx.Exec(query)
```

Comment:
No More dropping table. It will create if not exists and rest of the nodes will not impact the table. 
This is done to avoid any one time execution outside the automation.